omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -1046,6 +1046,24 @@
         </h1>
     </ifMode>
 
+    <script type="text/javascript">
+        // Hack to make "Confirmado" appear as "1a vez"
+        // See https://pihemr.atlassian.net/browse/MEX-459
+
+        $('body').on('DOMNodeInserted', 'div', function () {
+            const targetText = "1a vez";
+            const confirmedLabel = jq('label:contains("Confirmado")');
```

Comment:
Is there an id on this label?  Seems like there's a risk here if someone changes the display test to something other than "Confirmado", which I would think someone might assume they could do without causing any downstream changes?